apps/contrib/mixins.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+from django.utils import translation
+from django.utils.translation import gettext_lazy as _
+from wagtail.admin.panels import FieldPanel
+from wagtail.admin.panels import MultiFieldPanel
+from wagtailmetadata.models import MetadataPageMixin
+
+
+class TranslatedMetadataPageMixin(MetadataPageMixin, models.Model):
```

Comment:
Since we are basically overwriting everything from MetadataPageMixin here, I would rather inherit from WagtailImageMetadataMixin and also add the missing stuff here (i.e. the image and the methods).